### PR TITLE
Update doctor's comments on formula

### PR DESCRIPTION
**Pull request by javatarz**
Will help vpn-kill-switch/killswitch#20 when this repo gets more stars.

### CODE REVIEW

File: killswitch.rb [hunk 1]
```diff
@@ -1,9 +1,7 @@
-class Killswitch < Formula
-  desc "create & load a kill-switch pf.conf"
+class VpnKillSwitch < Formula
```
Comment (nbari) [inline killswitch.rb]: Could we keep the name `Killswitch`

**Review by nbari — CHANGES_REQUESTED** [review]

File: killswitch.rb (hunk 1 above)
Comment (javatarz) [reply killswitch.rb]: Had renamed when I built the homebrew formula on my local to pass the doctor test.

Renamed it to `Killswitch` again.

Comment (nbari) [reply killswitch.rb]: What is the "doctor test"?

**Comment by nbari** [comment]
@javatarz many thanks 👍

File: killswitch.rb (hunk 1 above)
Comment (javatarz) [reply killswitch.rb]: I meant `brew doctor` from [their contributing.md](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md)

**Pull request merged**